region/maridia/inner-pink/East Sand Hole.json

```diff
@@ -470,6 +380,41 @@
       "flashSuitChecked": true,
       "note": "The shinespark must be performed near the upper ledge, and not inside the sandfall."
     },
+    {
+      "link": [1, 4],
+      "name": "Use Flash Suit",
+      "requires": [
+        "Morph",
+        "canPlayInSand",
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 17, "excessFrames": 4}},
+        {"or": [
+          "HiJump",
+          "canSpringBallJumpMidAir",
+          "canSunkenTileWideWallClimb"
+        ]}
+      ],
+      "note": [
+        "Shinespark to the first ledge. To use a flash suit in deep sand:",
+        "vertically - quickly roll the directional inputs from forward to up and jump.",
+        "diagonally, with minimal horizontal speed - while holding angle, quickly tap forward then press and hold jump."
+      ]
+    },
+    {
+      "link": [1, 4],
+      "name": "Diagonal Shinespark, Use Flash Suit",
+      "entranceCondition": {
+        "comeInNormally": {},
+        "comesThroughToilet": "no"
+      },
+      "requires": [
+        "Morph",
+        "canPlayInSand",
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 19, "excessFrames": 7}}
+      ],
+      "note": "On entrance, get a good jump off of the sand then spark diagonally up and out of the water."
```

Comment:
I thought this meant enter the room spinning and use the flash suit on the way down.